content/prezent-za-zgloszenie.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Prezent za zgłoszenie"
+date: 2020-03-18T06:41:42+00:00
+lastmod: 2020-03-26T22:07:32+00:00
+draft: false
+author: "Łukasz Ziobroń"
+authorLink: ""
+description: ""
+summary: ""
+license: ""
+show_in_homepage: false
+show_description: false
+url: "/prezent-za-zgloszenie/"
+
+tags: []
+categories: []
+hiddenFromHomePage: true
+
+featuredImage: ""
+featuredImagePreview: ""
+
+comment: false
+toc: false
+autoCollapseToc: true
+math: false
+---
+
+<div style="text-align: center">
+
+# Dzięki za zgłoszenie! 🥳
+
+{{< image src="../static/assets/prezenty/minion_excited.gif" >}}
```

Comment:
```suggestion
{{< image src="/assets/prezenty/minion_excited.gif" >}}
```
Do katalogu static z tego co pamiętam odnosi się po prostu przez /. To co tam jest powinno być widoczne tak jakby było w katalogu głównym witryny. Ale jeszcze to sprawdzę u siebie.